db/migrate/20201215215458_create_nod_date_edit.rb

```diff
@@ -0,0 +1,13 @@
+class CreateNodDateEdit < Caseflow::Migration
+  def change
+    create_table :nod_date_edits do |t|
+      t.references "appeal", null: false, foreign_key: true, comment: "Appeal that NOD date is being edited for"
+      t.date "old_value", null: false, comment: "Value before update"
+      t.date "new_value", null: false, comment: "Value after update"
```

Comment:
```suggestion
      t.date "old_date", null: false, comment: "Date before update"
      t.date "new_date", null: false, comment: "Date after update"
```
Makes it clear in SQL queries that we are working with dates.